client/src/lib/components/DatasetListItem/DatasetListItem.svelte

```diff
@@ -16,104 +16,118 @@
+      </p>
+
+      <div class="metadata-items fr-mt-1w">
+        <div>
+          <span class="fr-fi fr-fi-x-map-2-line" aria-hidden="true" />
+          <p
+            class="fr-text--xs fr-my-0 fr-px-1w metadata-geographical-coverage"
           >
-        </p>
+            <span class="fr-text-mention--grey">Couverture géographique</span>
+            <br />
+            <span
+              >{GEOGRAPHICAL_COVERAGE_LABELS[
+                dataset.geographicalCoverage
+              ]}</span
+            >
+          </p>
+        </div>
+        <div>
+          <span class="fr-fi fr-fi-file-line" aria-hidden="true" />
+          <p class="fr-text--xs fr-my-0">
+            <span class="fr-text-mention--grey">Formats</span> <br />
+            <span>{formatFormats(dataset)}</span>
+          </p>
+        </div>
       </div>
-      <div>
-        <span class="fr-fi fr-fi-file-line" aria-hidden="true" />
-        <p class="fr-text--xs fr-my-0">
-          <span class="fr-text-mention--grey">Formats</span> <br />
-          <span>{formatFormats(dataset)}</span>
+      {#if dataset.headlines}
+        <p class="fr-mb-0 fr-mt-1w fr-text--sm fr-text-mention--grey">
+          <em data-testid="headlines-description"
+            >... {@html dataset.headlines.description} ...</em
+          >
         </p>
-      </div>
+      {/if}
     </div>
-    {#if dataset.headlines}
-      <p class="fr-mb-0 fr-mt-1w fr-text--sm fr-text-mention--grey">
-        <em data-testid="headlines-description"
-          >... {@html dataset.headlines.description} ...</em
-        >
+
+    <div class="actions">
+      <p class="fr-text--sm">
+        {capitalize(
+          formatDaysMonthsOrYearsToNow(dataset.catalogRecord.createdAt)
+        )}
       </p>
-    {/if}
-  </div>
 
-  <div class="actions">
-    <p class="fr-mb-2w fr-text--sm fr-text-mention--grey">
-      {capitalize(
-        formatDaysMonthsOrYearsToNow(dataset.catalogRecord.createdAt)
-      )}
-    </p>
-    <a
-      href={paths.datasetDetail({ id: dataset.id })}
-      class="fr-link fr-fi-arrow-right-line fr-link--icon-right"
-      title="Consulter cette fiche de données"
-    >
-      Voir
-    </a>
-  </div>
+      <span class="fr-fi-arrow-right-line" />
+    </div>
+  </a>
 </li>
 
 <style>
-  li {
-    display: grid;
-    grid-template-columns: auto 1fr auto;
-    column-gap: 2rem;
+  li,
+  a,
+  .container {
+    width: 100%;
   }
 
+  a {
+    display: flex;
+    width: 100%;
+  }
   li:not(:last-child) {
     border-bottom: 1px solid var(--border-default-grey);
   }
 
   .metadata-items {
     display: flex;
+    justify-content: space-between;
     flex-wrap: wrap;
     margin: 0 -2em;
+    width: 60%;
   }
 
   .metadata-items > * {
-    flex: 0 1 auto;
     margin: 0 2em;
-    display: grid;
-    grid-template-columns: auto 1fr;
-    align-items: center;
-    column-gap: 0.5rem;
+    display: flex;
   }
 
-  .metadata-items [class*="fr-fi"] {
+  [class*="fr-fi"] {
```

Comment:
Ceci s'applique désormais à l'icône anciennement dénotée "Voir" (la flèche sur la droite de la list item), ce qui la décale à l'intérieur de la ligne, contrairement à la maquette où elle est bien à droite. Il me semble qu'il ne faut appliquer ceci qu'aux icônes de metadata.

```suggestion
  .metadata-items [class*="fr-fi"] {
```

Et on peut ajouter la classe `fr-text-label--blue-france` à l'icône flèche pour la mettre en bleu (trouvée dans le `dist` du DSFR)